<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>eActionButton</class>
  <widget class="QDialog" name="eActionButton" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>484</width>
        <height>228</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Action button</string>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <widget class="Q3ButtonGroup" name="Action" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>3</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="midLineWidth" >
            <number>0</number>
          </property>
          <property name="title" >
            <string>Simple action</string>
          </property>
          <property name="checkable" >
            <bool>true</bool>
          </property>
          <property name="checked" >
            <bool>false</bool>
          </property>
          <property name="exclusive" >
            <bool>false</bool>
          </property>
          <property name="radioButtonExclusive" >
            <bool>false</bool>
          </property>
          <layout class="QGridLayout" >
            <item row="0" column="0" >
              <layout class="QGridLayout" >
                <item row="0" column="0" >
                  <widget class="QCheckBox" name="cbUpdate" >
                    <property name="text" >
                      <string>Update</string>
                    </property>
                  </widget>
                </item>
                <item row="3" column="0" >
                  <widget class="QCheckBox" name="cbClose" >
                    <property name="text" >
                      <string>Close</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="0" >
                  <widget class="QCheckBox" name="cbTurnon" >
                    <property name="text" >
                      <string>Turn on</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="Q3ButtonGroup" name="Script" >
          <property name="title" >
            <string>User action</string>
          </property>
          <property name="checkable" >
            <bool>true</bool>
          </property>
          <property name="checked" >
            <bool>false</bool>
          </property>
          <property name="exclusive" >
            <bool>false</bool>
          </property>
          <property name="radioButtonExclusive" >
            <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <layout class="QHBoxLayout" >
                <item>
                  <widget class="QLabel" name="textLabel1" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>5</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string>Execute</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QComboBox" name="cbAction" />
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item rowspan="2" row="0" column="1" colspan="1" >
        <layout class="QVBoxLayout" >
          <item>
            <widget class="QPushButton" name="pushButton1" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>OK</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="pushButton2" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>Cancel</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>121</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Vertical</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="5" margin="5" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local" >wactionbutton.h</include>
    <include location="local" >awidget.h</include>
  </includes>
  <connections>
    <connection>
      <sender>Action</sender>
      <signal>toggled(bool)</signal>
      <receiver>eActionButton</receiver>
      <slot>Action_toggled(bool)</slot>
    </connection>
    <connection>
      <sender>Script</sender>
      <signal>toggled(bool)</signal>
      <receiver>eActionButton</receiver>
      <slot>Script_toggled(bool)</slot>
    </connection>
    <connection>
      <sender>pushButton1</sender>
      <signal>clicked()</signal>
      <receiver>eActionButton</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>pushButton2</sender>
      <signal>clicked()</signal>
      <receiver>eActionButton</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
